Add installation details and Blender compatibility to README (#358)
godotengine/godot-blender-exporter: README.md

Native Godot scene format exporter for [Blender](https://www.blender.org), making the
export process to [Godot Engine](https://godotengine.org) as straightforward as possible.

**Warning:** This exporter is experimental and still lacks many features.

**Blender compatibility:** The current `master` branch requires at least Blender 2.80.

## Installation

1. Copy the `io_scene_godot` directory the location where Blender stores the
   scripts/addons folder on your system (you should see other io_scene_*
   folders there from other addons). Copy the entire dir and not just its

>    contents. For instance, on Linux, add-ons should be installed in `$HOME/.config/blender/<version>/scripts/addons` (create folders as needed). `<version>` should be replaced with your Blender version (such as `2.83` or `2.91` - don't include the letter or patch number if there is one). See [Blender directory layout](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html) for more information

Reviewer: ```suggestion
   contents. For instance, on Linux, add-ons should be installed in
   `$HOME/.config/blender/<version>/scripts/addons` (create folders as needed).
   `<version>` should be replaced with your Blender version (such as `2.83`
   or `2.91` - don't include the letter or patch number if there is one).
   See [Blender directory layout](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html)
   for more information.
```

2. Go to the Blender settings and enable the "Godot Exporter" plugin.

>    ![](install.png)

Reviewer: ```suggestion
   ![Enabling the add-on in the Blender user preferences](install.png)
```

Native Godot scene format exporter for [Blender](https://www.blender.org), making the
export process to [Godot Engine](https://godotengine.org) as straightforward as possible.

**Warning:** This exporter is experimental and still lacks many features.

**Blender compatibility:** The current `master` branch requires at least Blender 2.80.

## Installation

> 1. Copy the `io_scene_godot` directory the location where Blender stores the
>    scripts/addons folder on your system (you should see other io_scene_*
>    folders there from other addons). Copy the entire dir and not just its

Reviewer: ```suggestion
1. Copy the `io_scene_godot` directory the location where Blender stores the
   `scripts/addons` folder on your system. Copy the *entire* directory and not just its
```

export process to [Godot Engine](https://godotengine.org) as straightforward as possible.

> *WARNING*: This exporter is experimental, and still lacks many features.
> *WARNING*: The current master branch requires at least Blender 2.80

Reviewer: ```suggestion
**Warning:** This exporter is experimental, and still lacks many features.

**Blender compatibility:** The current `master` branch requires at least Blender 2.80.
```

## Installation

1. Copy the `io_scene_godot` directory the location where Blender stores the
   scripts/addons folder on your system (you should see other io_scene_*
   folders there from other addons). Copy the entire dir and not just its

>    contents. For instance, under Ubuntu for Blender 2.5+, addons are usually
>    in /usr/share/blender/scripts/addons
>    If you dont have sudo access, you can also install the plugin in your .config,
>    see the [Blender directory layout](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html)

Reviewer: Always recommend user-wide paths to reduce the risk of people breaking their system :slightly_smiling_face: 

```suggestion
   contents. For instance, on Linux, add-ons should be installed in `$HOME/.config/blender/<version>/scripts/addons` (create folders as needed). `<version>` should be replaced with your Blender version (such as `2.83` or `2.91` - don't include the letter or patch number if there is one). See [Blender directory layout](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html) for more information
```
